TODO

```diff
@@ -0,0 +1,19 @@
+https://github.com/ericmj/postgrex/blob/5037f45214cff04738ee3d359832443dfaa1494a/test/query_test.exs#L529
+=> "stream a reopened statement is ok"
+
+https://github.com/ericmj/postgrex/blob/5037f45214cff04738ee3d359832443dfaa1494a/test/query_test.exs#L539
+=> can re execute a closed query
+=> not sure I understood, current stream test does not bring anything new to query_test.exs#L539
```

Comment:
We want to try `prepare`, `close` and the  `stream` the prepared query. This should work so we can re-use queries prepared on other connections.